static/shortener.html

```diff
@@ -0,0 +1,145 @@
+            font-size: 15px;
+            color: #9e9e9e;
+        }
+         
+        input:-moz-placeholder { /* Firefox 18- */
+            font-weight: 600;
+            font-size: 15px; 
+            color: #9e9e9e;
+        }
+         
+        input::-moz-placeholder {  /* Firefox 19+ */
+            font-weight: 600;
+            font-size: 15px; 
+            color: #9e9e9e; 
+        }
+         
+        input:-ms-input-placeholder {  
+            font-weight: 600;
+            font-size: 15px; 
+            color: #9e9e9e; 
+        }
+
+        input[type=submit] {
+            background-color: #64b;;
+            border-radius: 2px;
+            border: none;
+            color: white;
+            font-size: 14px;
+            font-weight: 600;
+            padding: 15px 25px;
+            text-decoration: none;
+            text-transform: uppercase;
+            -webkit-transition: 0.5s;
+            transition: 0.5s;
+            cursor: pointer;
+        }
+
+        input[type=submit]:hover {
+            box-shadow: 0 6px 8px 0 rgba(0,0,0,0.24), 0 8px 25px 0 rgba(0,0,0,0.19);
+        }
+    </style>
+
+    <body>
+    	<div class="container">
+            
+            <div class="row">
+                <div class="col l10 offset-l1 m12 s12">
+                    <div class="card-panel">
+                        <h4 class="center header-text">Bel.ng url shortener</h4>
+                        <form id="shortURLForm">
+                            <div class="row">
+                                <div class="input-field col l9 s12">
+                                    <input type="text" name="url" id="id_url" 
+                                           placeholder="ENTER URL: MUST START WITH 'https://bel.ng/'">
+                                    </input>  
+                                </div>
+                                <span class="col l3" style="margin-top: 12px">
+                                    <input type="submit" value="get short url" id="id_short_url_btn"></input>
+                                </span>
+                            </div>
+                        </form>              
+                    </div>
+
+                    <div id="show-short-url">
+                        
+                    </div>
+                </div>
+            </div>
+        </div>
+    </body>
+
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.97.6/js/materialize.min.js"></script>
+    <script type="text/javascript">
+    	
+    	var form = document.getElementById("shortURLForm");
+        form.addEventListener("submit", function(event) {
+            event.preventDefault();
+            var divNode = document.getElementById("show-short-url");
+            var response = document.createTextNode("Loading...");
+            var longURL = document.getElementById('id_url').value;
+            var request = new XMLHttpRequest();
+            divNode.removeChild(divNode.childNodes[0])
```

Comment:
you don't have to create a text node manually. Just have existing div, and do `textDiv.textContent = newText`